tools/appveyor.psm1

```diff
@@ -342,6 +348,32 @@ function Invoke-AppVeyorTest
     Set-BuildVariable -Name TestPassed -Value True
 }
 
+function Compress-CoverageArtifacts
+{
+    # Create archive for test content, OpenCover module and CodeCoverage build
+    if(Test-DailyBuild)
```

Comment:
This if seems unneeded. Whoever call `Compress-CoverageArtifacts` should already know what they are doing.